### ISSUE TITLE
How can I multiply a weights matrix in Keras

### ISSUE THREAD

**Issue by FiammettaC** [keras-team/keras#10136]
Hello, I built a Bidirectional LSTM, with the following structure:

model = Sequential()
model.add(Embedding(vocab_size, embedding_size, input_length=55, weights=[pretrained_weights])) 
model.add(Bidirectional(LSTM(units=embedding_size)))
model.add(Dense(14146, activation='softmax')) 
#14146 because it's the total number of classes (different labels)
            
model.compile(loss='sparse_categorical_crossentropy',
              optimizer = RMSprop(lr=0.001),
              metrics=['accuracy'])
    
model.fit(np.array(X_train), np.array(y_train), epochs=100)

I would like to, before compiling the model, multiply the model's weights by a fixed matrix of 0 and 1.
I read that I could use something like this:
Matrix_Layer = Conv1D(filters=num_classes, kernel_size=num_classes, use_bias=False, strides=1, name='Matrix_Layer')
model.add(Matrix_Layer)

but I don't understand how to load my matrix and which weights to multiply to! My matrix is 55x64284, 
where 64284 corresponds to sentences and 55 is the max_lenght of the sentences (the shorter ones have been padded). It is made of 0 and 1, where the 1 is placed if the word in the sentence is a verb. 
Example:
I go to the cinema
0 1 0 0 0 
The result that I want to achieve is to multiply the matrix to the weights of the model, to restrict the vocabulary of the LSTM to only verbs. My X are sentences where the verbs have been extracted and the labels are the extracted verbs.
Example:
X : I _ to the cinema
y : go

I hope it makes sense!
Thanks in advance!
